src/ensembl/src/content/app/browser/browser-tabs/BrowserTabs.tsx

```diff
@@ -22,40 +26,43 @@ const BrowserTabs: FunctionComponent<BrowserTabsProps> = (
       return;
     }
 
+    if (!props.isTrackPanelOpened) {
+      props.toggleTrackPanel(true);
+    }
+
     if (props.isDrawerOpened) {
       props.closeDrawer();
     }
 
     props.selectBrowserTabAndSave(value);
   };
 
-  const getBrowserTabClasses = (trackType: TrackType) => {
-    const {
-      isDrawerOpened,
-      selectedBrowserTab,
-      isTrackPanelModalOpened
-    } = props;
-    let classNames = styles.browserTab;
-
+  const isBrowserTabActive = (trackType: TrackType) => {
     if (
+      props.isTrackPanelOpened &&
       props.ensObject.genome_id &&
-      selectedBrowserTab === trackType &&
-      !isDrawerOpened &&
-      !isTrackPanelModalOpened
+      props.selectedBrowserTab === trackType &&
+      !props.isDrawerOpened &&
+      !props.isTrackPanelModalOpened
     ) {
-      classNames += ` ${styles.browserTabActive} ${styles.browserTabArrow}`;
-    } else if (!props.ensObject.genome_id) {
-      classNames = styles.browserTabDisabled;
+      return true;
+    } else {
+      return false;
     }
```

Comment:
Same as comment above: the condition checks whether something is true and then returns true. Why not just return the result of the boolean check that is the condition itself?

It's been the subject of various memes, e.g.:

![image](https://user-images.githubusercontent.com/6834224/62065340-aea41100-b226-11e9-9788-62858e616fd1.png)

Reply:
It is the same here too. There are quite a lot of evaluations that might make it difficult to find where the true and false statements are in the ternary operator 😄

Reply:
As I mentioned above, I'll return without any if statements and ternary operators.